pkg/scheduler/scheduler_test.go

```diff
@@ -694,7 +763,8 @@ func TestSchedulerWithVolumeBinding(t *testing.T) {
 
 			eventReason: "Scheduled",
 		},
-		"bound,invalid-pv-affinity": {
+		{
+			name: "bound,invalid-pv-affinity",
```

Comment:
The commas could probably be replaced with slashes in these test names. (I'm assuming you already know how `-run` interacts with subtests and subsubtests.)